testing/nginx-naxsi/nginx.initd

```diff
@@ -1,67 +1,40 @@
 #!/sbin/openrc-run
+supervisor=supervise-daemon
 
 description="Nginx http and reverse proxy server"
 extra_started_commands="reload reopen upgrade"
 
 cfgfile=${cfgfile:-/etc/nginx/nginx.conf}
-pidfile=/run/nginx/nginx.pid
+pidfile=/run/nginx/$SVCNAME.sd.pid
 command=/usr/sbin/nginx
 command_args="-c $cfgfile"
+command_args_foreground='-g "daemon off;"'
 required_files="$cfgfile"
 
 depend() {
-	need net
```

Comment:
`need net` breaks `lxc` containers if they set an ip address in their `config`

If you set an ip address in an `lxc` config the container does not need to start networking